tests/core/persistent-journal-namespace/task.yaml

```diff
@@ -0,0 +1,61 @@
+summary: Test journal.persistent core config option is honored by journal namespaces
+
+# these systems do not support journal quota groups due to their old systemd versions.
+# requires systemd v245+
+systems:
+  - -ubuntu-core-16-*
+  - -ubuntu-core-18-*
+
+prepare: |
+  systemctl status snapd > before.txt
+  "$TESTSTOOLS"/snaps-state install-local test-snapd-journal-quota
+  snap set system experimental.quota-groups=true
+  tests.cleanup defer snap unset system experimental.quota-groups
```

Comment:
nitpic: I would find it more logical to do this in the `restore` step

Reply:
While I see your point this is the pattern we're using around the spread tests for this kind of operation. I'll leave it as is